.github/workflows/jdl.yml

```diff
@@ -42,24 +42,30 @@ jobs:
     runs-on: ${{ matrix.os }}
     defaults:
       run:
+        shell: bash
         working-directory: ${{ github.workspace }}/app
     if: github.event.action != 'closed'
     timeout-minutes: 90
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macos-11]
+        os: [ubuntu-20.04, macos-11, windows-2022]
         suite:
           - postgresql-mvc
           - postgresql-webflux
         include:
-          - suite: postgresql-mvc
-          - suite: postgresql-webflux
+          - os: ubuntu-20.04
+            native-build-args: --verbose -J-Xmx10g
+          - os: macos-11
+            native-build-args: --verbose -J-Xmx13g
+          - os: windows-2022
+            native-build-args: --verbose -J-Xmx10g
```

Comment:
Why do these need to be different? In my experience, `-J-Xmx10g` works for all of them.

Reply:
macOS runner has 16G, while Linux and windows has 8g.
So it may improve the build time.